modules/backend/widgets/Search.php

```diff
@@ -41,6 +41,11 @@ class Search extends WidgetBase
      */
     public $scope;
 
+    /**
+     * @var bool Search on every key stroke.
+     */
+    public $everykey = true;
```

Comment:
Maybe would be better to rename it to "onEveryKey" or "fireOnEveryKey".

Reply:
Cool - I will amend my pull request.

Reply:
Actually it was backwards compatible - the `everykey` option defaulted to true. But I also think that making it the other way around "sounds" more backward compatible, so I made a new Pull Request #3524